src/stories/components/loader/ProgressBar.stories.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import ProgressBar from "../../../../lib/components/Loader/ProgressBar";
+
+export default {
+  title: "Components/ProgressBar/Default",
+  component: ProgressBar,
+  argTypes: {
+    size: {
+      options: ["small", "medium", "large", "xLarge"],
+      control: { type: "radio" },
+    },
+  },
+};
+
+const Template = (args) => <ProgressBar {...args} />;
+
+export const Default = Template.bind();
+
+Default.args = {
```

Comment:
It is perfectly fine as is, but if you do not like the theme arg showing as a control on storybook, you can remove it using this technique: https://storybook.js.org/docs/react/essentials/controls#filtering-controls

but you can leave as is, most of the others are the same, storybook gets all props automatically and does this.